recipes/libcurl/all/conanfile.py

```diff
@@ -535,12 +535,10 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "CURL"
-        self.cpp_info.names["cmake_find_package_multi"] = "CURL"
-        self.cpp_info.names["pkg_config"] = "libcurl"
-        self.cpp_info.components["curl"].names["cmake_find_package"] = "libcurl"
-        self.cpp_info.components["curl"].names["cmake_find_package_multi"] = "libcurl"
-        self.cpp_info.components["curl"].names["pkg_config"] = "libcurl"
+        self.cpp_info.set_property("cmake_file_name", "CURL")
+        self.cpp_info.set_property("cmake_target_name", "CURL")
+        self.cpp_info.set_property("pkg_config_name", "libcurl")
+        self.cpp_info.components["curl"].set_property("cmake_target_name", "libcurl")
```

Comment:
Not sure, but is there 

```suggestion
        self.cpp_info.components["curl"].set_property("cmake_target_name", "libcurl")
        self.cpp_info.components["curl"].set_property("pkg_config_name", "libcurl")
```

missing?

Reply:
yes, the idea is to not create unwanted `curl.pc` file.

`cmake_target_namespace` property would allow to throw away component in this recipe, but it doesn't work yet for cmake_find_package and cmake_find_package_multi.